app/Resources/views/full/bundle_list.html.twig

```diff
@@ -6,14 +6,57 @@
             <div class="container">
                 <div class="row">
                     <div class="col-xs-12">
-                        {{ ez_render_field(content, 'description') }}
+                        <h1>{{ ez_render_field(content, 'name') }}</h1>
+                        <h2>{{ ez_render_field(content, 'description') }}</h2>
                     </div>
                 </div>
             </div>
         </div>
         <div class="bundles-list-content">
             <div class="container">
-                <div class="col-xs-12 bundles-list-container">
+                <div class="hidden-xs hidden-sm col-md-2 bundle-list-settings">
+                    <h2>Categories</h2>
+                    <ul class="bundle-list-settings-list">
+                        <li class="active"><a href="#">All Bundles</a></li>
+                        <li><a href="#">Content Type</a></li>
+                        <li><a href="#">Field Type</a></li>
+                        <li><a href="#">Theme</a></li>
+                        <li><a href="#">Social</a></li>
+                        <li><a href="#">e-Commerce</a></li>
+                        <li><a href="#">Administration</a></li>
+                        <li><a href="#">Workflow</a></li>
+                    </ul>
+                    <h2>Compatibility</h2>
```

Comment:
You can remove the `Compatibility` section. The first stable for v2 is planned for fall 2017. We might implement this filter in 2018.